Packages/com.unity.input/Runtime/Input.cs

```diff
@@ -647,27 +648,125 @@ public static bool GetKeyUp(KeyCode key)
 
         public static bool isGyroAvailable => InputSystem.Gyroscope.current != null;
 
-        #region Unimplemented
+        private static TouchPhase ToLegacy(InputSystem.TouchPhase phase)
+        {
+            switch (phase)
+            {
+                case InputSystem.TouchPhase.Began: return TouchPhase.Began;
+                case InputSystem.TouchPhase.Moved: return TouchPhase.Moved;
+                case InputSystem.TouchPhase.Ended: return TouchPhase.Ended;
+                case InputSystem.TouchPhase.Canceled: return TouchPhase.Canceled;
+                case InputSystem.TouchPhase.Stationary: return TouchPhase.Stationary;
+                    //InputSystem.TouchPhase.None  // These are filtered out and shouldn't be seen
+            }
+            throw new ArgumentOutOfRangeException(nameof(phase), $"Not expected phase value: {phase}");
+        }
```

Comment:
Makes me wonder whether we should go and unify the two. We could add `None` to the trunk side version and kill the `InputSystem` side `TouchPhase`. Binary breaking change but just seems confusing we not only have two such enums now but are actively using both.

Anyway, not directly related to this PR. Just something for us to consider, I think.